### PR TITLE
Update federated testing docs.

### PR DESCRIPTION
**Pull request by spxtr**
See #780. Unfortunately this whole system is in transition, so it won't be accurate for long. cc @chuckbutler

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/1013)

### CODE REVIEW

File: docs/federated_testing.md [hunk 1]
```diff
@@ -3,24 +3,25 @@
...
 To contribute test results for a platform, an organization executes the
 e2e test cases and uploads the test results to a Google Cloud Storage
-(GCS) bucket.  The Kubernetes project contains scripts in `hack/` to run
-the tests and to upload the results.
-
-The test history dashboard is generated periodically by scripts in this
-repository that read test results from all of the GCS buckets.  The test
-history shows test results from jobs run during the previous 24 hours.
-A test job should execute at least once every 24 hours.
+(GCS) bucket.  The test-infra repo contains scripts in `jenkins/` to run the
```
Comment (rmmh) [inline docs/federated_testing.md]: give a direct link to test-infra

**Comment by rmmh** [comment]
Woops, thought we were in kubernetes/kubernetes.​

**Comment by chuckbutler** [comment]
` hack/jenkins/upload-to-gcs.sh`  appears to only exist in kubernetes/kubernetes though. is that script making the move to test-infra?

**Comment by spxtr** [comment]
Soon, yes. I think we're making it happen in `jenkins/bootstrap.py`.

**Comment by chuckbutler** [comment]
@spxtr sorry to keep this thread goign, but its got a lot of good info here.  Question:  Is there a chance we can de-couple the test-run and munge/upload steps? I'm evaluating using just the `upload-to-gce.sh` script, as we're running the e2e tests out of band of the jenkins job.

All jenkins is doing is coordinating the test plan, and allowing us to pass through parameters to the job. We're running e2e in the cloud thats under load as a minimal unit, related to our k8s cluster.

I'm happy to either diagram, hop on a hangout, or otherwise describe the process soup to nuts as we have it outlined today. I'm feeling pretty good about where we're headed and it seems like keeping the running/result-collection as separate steps would be a good thing for our approach.

I'm happy to maintain any forks of what we're doing, but I'll need to get more involved with the testing sig so i make sure we dont drift.

**Comment by spxtr** [comment]
Lets continue in #780.

**Pull request merged**